static/boosthub-preload.js

```diff
@@ -78,7 +78,8 @@ window.__ELECTRON_ONLY__.openInBrowser = openInBrowser
 
 const handler = (event) => {
   event.preventDefault()
+  event.stopPropagation()
   sendToHost('open-context-menu')
 }
 window.addEventListener('contextmenu', handler)
-window.__ELECTRON_ONLY__.globalContextMenuIsConfigured
+window.__ELECTRON_ONLY__.globalContextMenuIsConfigured = true
```

Comment:
@Rokt33r you forgot to set this to true, because this would be invalid JS to leave it just specified?
Is it fine to set it to `true`?

Reply:
LGTM! Thanks!